en/controllers/components/authentication.rst

```diff
@@ -1056,6 +1056,8 @@ unauthorizedRedirect
 storage
     Storage class to use for persisting user record. When using stateless
     authenticator you should set this to ``Memory``. Defaults to ``Session``.
+    You can pass config options to storage class using array format. For e.g. to
+    use a custom session key you can set ``storage`` to ``['className' => 'Storage', 'key' => 'Auth.Admin']``.
```

Comment:
Would it make sense to place a ref/hint where to continue reading if you want to create your own Storage adapter classes?